<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgSeExprSavePreset</class>
 <widget class="QWidget" name="WdgSeExprSavePreset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>332</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Save SeExpr Preset</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>13</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="presetNameLayout">
     <item>
      <widget class="QLabel" name="newPresetNameLabel">
       <property name="text">
        <string>Preset name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="newPresetNameTextField">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="presetThumbnailManagementLayout">
     <property name="spacing">
      <number>18</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="presetThumbnailLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="presetThumbnailWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>200</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="presetLoadThumbnailLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="loadExistingThumbnailButton">
         <property name="text">
          <string>Load Existing Thumbnail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadImageIntoThumbnailButton">
         <property name="text">
          <string>Load Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadRenderFromScriptButton">
         <property name="text">
          <string>&amp;Render Script to Thumbnail</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearPresetThumbnailButton">
         <property name="text">
          <string>Clear Thumbnail</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
